manifests/crd.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-# This manifest is auto-generated from the helm chart, do not modify! 
+# This manifest is auto-generated from the helm chart, do not modify!
```

Comment:
You need to add the changes only in `helm-chart/amalthea/crd-spec.yaml`, the `crd.md` in docs and the `crd.yaml` in `manifests` will be automatically updated with an automatically generated PR.